code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -567,7 +567,7 @@ SUBSYSTEM_DEF(dynamic)
 /datum/controller/subsystem/dynamic/proc/post_setup(report)
 	for(var/datum/dynamic_ruleset/roundstart/rule in executed_rules)
 		rule.candidates.Cut() // The rule should not use candidates at this point as they all are null.
-		addtimer(CALLBACK(src, TYPE_PROC_REF(/datum/controller/subsystem/dynamic/, execute_roundstart_rule), rule), rule.delay)
+		addtimer(CALLBACK(src, PROC_REF(execute_roundstart_rule), rule), rule.delay)
```

Comment:
Why this one?

Reply:
src is `/datum/controller/subsystem/dynamic`, TYPE_PROC_REF is unnecessary there because the proc is defined on src. it just got batched in with my other changes, it's not technically necessary but it reads as cleaner to me